lib/modules/manager/bundler/extract.ts

```diff
@@ -16,12 +16,70 @@ export async function extractPackageFile(
   content: string,
   packageFile?: string,
 ): Promise<PackageFileContent | null> {
+  let lineNumber: number;
+  async function processGroupBlock(
```

Comment:
it's still nested. why not simply passing line number as argument and return the new values. we do that on other managers

Reply:
I looked again and really feel it's not worth the refactoring because it also mutates res.deps